execution/executor/src/block_processor.rs

```diff
@@ -52,6 +52,9 @@ pub(crate) struct BlockProcessor<V> {
     /// The timestamp of the last committed ledger info.
     committed_timestamp_usecs: u64,
 
+    /// The last committed version
+    committed_version: Option<Version>,
```

Comment:
Do you need this field? This info can be derived from `self.committed_transaction_accumulator`.

Reply:
resolved.